README.md

```diff
@@ -36,10 +51,15 @@ sudo apt-get install python3-pip
 ### Yaml-cpp
 
 If this software package is not available in the system (it is shipped with some software, e.g. ROS), dowload it and install it from <https://github.com/jbeder/yaml-cpp>. **Please make sure it is installed in a share library mode**.
+Klepsydra [SDK tutorial](https://github.com/klepsydra-technologies/kpsr-tutorial/blob/main/README.md) will require yaml support.
+
+```bash
+sudo apt-get install libyaml-cpp-dev
```

Comment:
this approach works only for the case of ROS. otherwise there are too many issues with yaml-cpp. The correct instruction would be to compile and install v0.7.0 from source, using the cmake options `-DCMAKE_POSITION_INDEPENDENT_CODE=ON -DYAML_CPP_BUILD_TESTS=Off`

Reply:
Is it worth adding this info to the doc?